### ISSUE TITLE
Original texture is almost completely overpowered

### ISSUE THREAD

**Issue by timsegger** [OneOnlineShow/F76CIG#1]
I tried several options in the colors.txt

00FF00 and 000100 as colorcodes produce the same color
And reducing either or both "Emittance Multiplier" and "Adaptive Exposure Effect" to anything above 0.00 will not reduce the color intensity either.

How would I achieve a just slightly greener texture to increase contrast?

**Comment by oosname** [comment]
Hmm, maybe it depends on the model and its texture, have you tried different ones?  As far as I remember it worked well, it was possible to get a bright or dull color using these settings... maybe something has changed in F76, but I'm currently working on another project.
Make sure the changes are applied, change the color to red for example.  Try to make your own texture, maybe it is required, I don't remember.  Play with both parameters, make them greater than 0, then one 0 and the other 1 and vice versa (with/without active custom texture). Try values ​​greater than 1 (10/100).

**Comment by timsegger** [comment]
I downloaded the FO4 Material Editor to examine the files that were generated and found out that when I entered an "Emittance Multiplier" of "0.0500" it was read as "500x".

When I used "0,0500" it was correctly interpreted. I guess your Tool requires the decimal seperator to be localized. In my country we use ",".

But I also found that other "Glow"-Mods use the items "Diffuse"-Material as Glow Material and do not use "Adaptive Emissive".
So I will try to change your tool accordingly.
